README.md

```diff
@@ -197,3 +197,18 @@ The UI will only be available for the duration specified by the `--ui-runtime`.
 default this is 10 minutes. After this time, the web server will be disabled. The 
 application must be restarted to re-enable the UI. You can set this to `0` to 
 disable the timeout. This is not recommended.
+
+## Development
+
+### Scripts
+
+The following scripts are available:
+
+ - `npm start` - Start the agent
+ - `npm run dev` - Build the agent and watch for changes
+ - `npm run update-theme` - Update the theme files from the nr-launcher repository
```

Comment:
This probably needs adding to the build build GH Action to ensure it always has the latest.

Also need to check the admin/build scripts and order in the release issue to ensure things are done in the right order.

Reply:
it does and is raised as a task.